### DISCUSSION TITLE
Mixamo Animation Bug in Spoke

### DISCUSSION THREAD

**Discussion by TharHtetAungg** [mozilla/hubs#6047] [Spoke Questions]
Hi, my spoke environment has a bug when I apply the animation model (make with Mixamo) the hub starts to change white blank wall, and all of the models are gone! But when I remove the animation from the model, it's working.

![image](https://user-images.githubusercontent.com/79152768/230610103-3e9ee57f-317c-4c0e-9291-ec5b1bcc250d.png)

**Comment by cvanmeurs** [comment]
If an animated mesh is disappearing, there's a good chance it is due to frustum culling.  Hubs likely thinks the object is not where it actually is due to being a skinned mesh.  In Blender you fix this by adding a "Frustum" component and un-checking "culled".  I'm not sure how that is handled in Spoke.
